.github/workflows/documentation.yml

```diff
@@ -23,12 +23,21 @@ name: Documentation
 
 on:
   push:
+    paths-ignore:
+      - 'docs/**'
+      - '*.md'
+      - '*.html'
+      - 'src/main/python/docs/**'
+      - 'dev/**'
     branches:
       - master
   pull_request:
     paths-ignore:
       - 'docs/**'
       - '*.md'
+      - '*.html'
+      - 'src/main/python/docs/**'
+      - 'dev/**'
```

Comment:
This is the one tests i would run on changes in docs, since it is testing if the docs build.

Reply:
Is this the change?

```diff
  pull_request:
    paths-ignore:
      - 'docs/**'
      - '*.md'
      - '*.html'
-      - 'src/main/python/docs/**'
      - 'dev/**'
```

or should not ignore any paths here?


Thanks

Reply:
'docs/**'
 'src/main/python/docs/**'
 and
 '*.md'
should not be ignored when we test building the docs.